packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -689,7 +689,13 @@ class RCTAztecView: Aztec.TextView {
     ///
     private func refreshFont() {
         let newFont = applyFontConstraints(to: defaultFont)
+        font = newFont
```

Comment:
It's really interesting that this didn't take place before. I also see remnants of a past history where `refreshFont` probably called `refreshTypingAttributesAndPlaceholderFont`, due to the comment on line 702. 🤷